Portfolio/index.css

```diff
@@ -1 +1,88 @@
-/* Write your code below for PORTFOLIO */
+/* Write your code below for PORTFOLIO */
+*{
+  
+    margin:  0px;

```

Comment:
The value for 'padding' is missing units (e.g., px, em). Add a unit to the value, such as '10px'.